README.md

```diff
@@ -202,6 +202,10 @@ color_thresholds:
   - value: 4
     color: "#0000ff"
 ```
+i.e. if value is between "0" (including this value) & "1.33333" - color is "#ff0000",
+between "1.33333" (including this value) & "2.666667" - color is "#ffff00",
+between "2.666667" (including this value) & "4" - color is "#00ff00",
+equal to or more than "4" - color is "#0000ff".
```

Comment:
I think, this change should make the resulting colors more readable:

```suggestion
The example above will result in the following colors of the graph: if value is 
* between `0` (including this value) and  `1.33333`, the color is `#ff0000`,
* between `1.33333` (including this value) and `2.666667`, the color is `#ffff00`,
* between `2.666667` (including this value) and `4`, the color is `#00ff00`,
* equal to or more than `4`, the color is `#0000ff`.
```

What do you think? (If I got it right, you can just "apply" or "accept" the suggestion, if you agree.)

Reply:
Good, I think

Reply:
> fix the PR title 

Done!